apps/src/templates/instructions/teacherFeedback/TeacherFeedback.jsx

```diff
@@ -168,35 +187,38 @@ export class TeacherFeedback extends Component {
     );
   }
 
+  getLatestReviewState() {
+    const {latestFeedback} = this.state;
+    const isAwaitingTeacherReview =
+      latestFeedback?.review_state === ReviewStates.keepWorking &&
+      latestFeedback?.student_updated_since_feedback;
+    const reviewState = isAwaitingTeacherReview
+      ? ReviewStates.awaitingReview
```

Comment:
I forget, could you remind me if ReviewStates is just a front-end concept?  (I'm worried that it might be confused with the back-end review state which does not include the awaitingReview value, right?)

Reply:
ReviewStates is a front and backend concept with one slight difference, on the backend it can only have values `keepWorking` or `completed`, on the frontend it could also be `awaitingReview`

Reply:
Maybe add a comment where it's defined on the front-end to explain that awaitingReview is a derived state that's not stored in the database on the back-end?  (Could you remind me why didn't just have the back-end calculate this and send awaitingReview?  I remember talking about it but don't remember the reason.)

Reply:
Oh! I think the reason was because awaitingReview only really makes sense for the latest feedback so I couldn't generically have a TeacherFeedback.awaitingReveiw? method on the backend because it would have to be in context with other teacher feedbacks

Reply:
That's probably worth leaving a comment about somewhere, maybe where it's derived.

Reply:
Or (sorry for all the comments, I'm having more ideas here) maybe we could update the summarize method for teacher_feedbacks to take a parameter `isLatest`, if it was true would calculate `awaitingReview`. So then it could be calculated on the backend but a developer would know it only makes sense for the latest TeacherFeedback

Reply:
I think it's be worthwhile to try that last idea (in a separate PR).